the-collab-lab/tcl-45-smart-shopping-list
# Replace favicon with Pac-Man, change app title

## Description

The app now has its own favicon with Pac-Man and its title now reflects the name in the logo.

## Related Issue



Closes #48 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before

<img width="249" alt="Screenshot of favicon with React symbol before the changes" src="https://user-images.githubusercontent.com/63440229/186575993-cccb2d7b-5ba1-4123-a541-83f42f48f46d.png">

### After

<img width="280" alt="Screenshot with Pac-Man favicon and new title after the changes" src="https://user-images.githubusercontent.com/63440229/186576047-7f882e19-4a9b-4891-8fea-1702d48ccd8d.png">

## Testing Steps / QA Criteria

Checkout the branch `NZ-change-icon` and run `npm start`. In your browser tab you should see the new favicon with Pac-Man and the title should say 'Shop-Ade smart shopping application'
